tasks/build/css.js

```diff
@@ -99,7 +99,12 @@ function pack(type, compress) {
     concatenatedCSS = compress ? filenames.concatenatedMinifiedCSS : filenames.concatenatedCSS;
   }
 
-  return gulp.src(output.uncompressed + '/**/' + globs.components + ignoredGlobs)
+  let src = output.uncompressed + '/**/' + globs.components + ignoredGlobs;
+  if (globs.components.indexOf('table') < 0 && globs.components.indexOf('tab') > 0) {
```

Comment:
```suggestion
  if (globs.components.indexOf('table') < 0 && globs.components.indexOf('tab') >= 0) {
```
Just in case `tab` is a prefix of components.

Reply:
I'm not exactly sure what you mean with a prefix of components... The comparison is purely to see if the word 'tab' is present in the list, and 'table' is not.

indexOf will return the start position of the substring. So even if tab is the first component in the list, the output of indexOf is already 1:
```
{tab,reset,site,button,container,divider,header,icon,image,input,label,list,segment,text,breadcrumb,grid,menu,message,card,feed,item,statistic,accordion,checkbox,dropdown,embed,popup,rating,sidebar,sticky,transition,visibility}
1
```
Probably because of the { at the beginning.

Reply:
It would be much better if the components list is processed as a proper array of course, but my JS-fu is too limited to make that happen (quickly).

Reply:
> Probably because of the { at the beginning.

Ah, I see.